modern/src/Theme.js

```diff
@@ -0,0 +1,50 @@
+import { createMuiTheme } from '@material-ui/core/styles';
+
+const traccarPurple = '#333366';
+const traccarGreen = '#4CAF50';
+const traccarWhite = '#FFF';
```

Comment:
Can we use material colors?

Reply:
Here, I just copied the color code from the design file.
Should I replace these with material colors?

Reply:
I have added material colors for example.